I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label fail is defined twice, at
  File "game/fail.rpy", line 1 and
  File "game/fail_branch.rpy", line 1.

Ren'Py Version: Ren'Py 6.99.12.3.2123
